app/Infrastructure/Console/Commands/MoveUserDataToProfile.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Infrastructure\Console\Commands;
+
+use Illuminate\Console\Command;
+use Toby\Eloquent\Models\User;
+
+class MoveUserDataToProfile extends Command
+{
+    protected $signature = "toby:move-user-data-to-profile";
+    protected $description = "Move user data to their profiles";
+
+    public function handle(): void
+    {
+        $users = User::all();
+
+        /** @var User $user */
```

Comment:
Do we need this docblock with these here https://github.com/illuminate/collections/blob/master/Collection.php#L60 ?